src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html

```diff
@@ -13,120 +13,126 @@
-
-    <ng-container matColumnDef="heterologous_reactions">
-      <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Heterologous Reactions</span>
-        <div class="custom-slider">
-          <ng5-slider [options]="options['reactions']" [formControl]="reactionsFilter"></ng5-slider>
-        </div>
-      </th>
-      <td mat-cell *matCellDef="let element">
-        <a>{{element.heterologous_reactions.length}}</a>
-      </td>
-    </ng-container>
-
-    <ng-container matColumnDef="product">
-      <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Product</span>
-        <div class="custom-slider">
-          <ng5-slider [options]="options['product']" [formControl]="productFilter"></ng5-slider>
-        </div>
-      </th>
-      <td mat-cell *matCellDef="let element"> {{element.product | float}}</td>
-    </ng-container>
-
-    <ng-container matColumnDef="biomass">
-      <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Biomass</span>
-        <div class="custom-slider">
-          <ng5-slider [options]="options['biomass']" [formControl]="biomassFilter"></ng5-slider>
-        </div>
-      </th>
-      <td mat-cell *matCellDef="let element"> {{element.biomass | float}}</td>
-    </ng-container>
-
-    <ng-container matColumnDef="yield">
-      <th mat-header-cell *matHeaderCellDef><span mat-sort-header>Yield</span>
-        <div class="custom-slider">
-          <ng5-slider [options]="options['yield']" [formControl]="yieldFilter"></ng5-slider>
-        </div>
-      </th>
-      <td mat-cell *matCellDef="let element"> {{element.yield | float}}</td>
-    </ng-container>
-
-    <ng-container matColumnDef="fitness">
-      <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Fitness</span>
-        <div class="custom-slider">
-          <ng5-slider [options]="options['fitness']" [formControl]="fitnessFilter"></ng5-slider>
-        </div>
-      </th>
-      <td mat-cell *matCellDef="let element"> {{element.fitness | float}}</td>
-    </ng-container>
-
-    <ng-container matColumnDef="method">
-      <th mat-header-cell *matHeaderCellDef>
-        <span mat-sort-header>Method</span>
-        <mat-form-field class="example-full-width">
-          <input class="method-input" matInput [formControl]="methodFilter">
-        </mat-form-field>
-      </th>
-      <td mat-cell *matCellDef="let element"> {{element.method}}</td>
-    </ng-container>
-
-    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-    <tr mat-row
-        *matRowDef="let row; columns: displayedColumns;"
-        matRipple
-        class="element-row"
-        [appJobResultsDetailRow]="row"
-        [appJobResultsDetailRowTpl]="tpl"
-        (toggleChange)="toggleChange($event)">
-    </tr>
-  </table>
-</mat-card>
+<mat-error *ngIf="tableData.length === 0; else results">
+    There are no pathways to <strong>{{productName}}</strong> in <strong>{{organism$ | async}}</strong>.
```

Comment:
Why organism$ is an observable?

Reply:
I changed the code as it was before with getting organism name by its id.